code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -596,7 +599,8 @@
 	dancer.sound_environment_override = NONE
 
 /datum/reagent/drug/blastoff/on_mob_life(mob/living/carbon/dancer, seconds_per_tick, times_fired)
-	. = ..()
+	. = TRUE
+	..()
```

Comment:
```suggestion
	. = ..()
	return TRUE
```

Reply:
This is logically different from the current, though I have updated it to follow the style you have described.